server/services/booking.js

```diff
@@ -133,8 +144,21 @@ const bookingServiceFactory = (eliteApi, keyworkerApi) => {
         : null
     }
 
+    const getPrisonOffenderManagerName = pom => {
+      console.log(pom)
```

Comment:
if you still need the logging might be best to use logger.info or logger.debug

Reply:
Good spot! Have updated